driver-core/src/test/java/com/datastax/driver/core/WarningsTest.java

```diff
@@ -34,8 +34,7 @@ public void onTestContextInitialized() {
   @CassandraVersion("2.2.0")
   public void should_expose_warnings_on_execution_info() throws Exception {
     // the default batch size warn threshold is 5 * 1024 bytes, but after CASSANDRA-10876 there must
-    // be
-    // multiple mutations in a batch to trigger this warning so the batch includes 2 different
+    // be multiple mutations in a batch to trigger this warning so the batch includes 2 different
```

Comment:
:+1: this is probably a relic of running google-java-format initially.  Since multi-line comment format isn't used here, it doesn't know to correlate the following lines.